packages/components/src/components/Cell/Cell.tsx

```diff
@@ -75,6 +76,24 @@ export const Cell = forwardRef<View, React.PropsWithChildren<CellProps>>(
         const swipeable = !!leftSwipeGroup || !!rightSwipeGroup;
 
         const CellContent = () => (
+            <>
+                <View style={styles.contentContainer}>
+                    {leftAdornment && <View style={styles.adornment}>{leftAdornment}</View>}
+
+                    <View style={styles.children}>
+                        <View style={{ flex: 1, justifyContent: "center" }}>{children}</View>
+                    </View>
+                    {rightAdornment && <View style={styles.adornment}>{rightAdornment}</View>}
+                </View>
+                {!isLastCell && (
+                    <View style={styles.dividerOuter}>
+                        <View style={styles.dividerInner} />
+                    </View>
+                )}
+            </>
+        );
+
+        const Cell = () => (
```

Comment:
Having a component inside the Cell component with the same name can be a little bit confusing. Maybe find a better suited name for it?